### ISSUE TITLE
Zero value Audio would cause NAN problem

### ISSUE THREAD

**Issue by Eric-Chen-007** [SeanNaren/deepspeech.pytorch#272]
I find if there is an audio have zero value (for example, 3 seconds audio, its value is always 0),  it would get a NAN loss value.

           `loss = criterion(out, targets, sizes, target_sizes)
            loss = loss / inputs.size(0) 

            loss_sum = loss.data.sum()
            inf = float("inf")
            if loss_sum == inf or loss_sum == -inf:
                print("WARNING: received an inf loss, setting loss value to 0")
                loss_value = 0
            else:
                loss_value = loss.data[0]`

loss_value would equal "nan".

I find this problem because i use "voice.mozilla" dataset.
There are some zero value audio in this dataset.
Of course these zero value audios should be removed.
